src/layman/common/redis.py

```diff
@@ -80,6 +80,14 @@ def solve_locks(workspace, publication_type, publication_name, error_code, metho
     elif current_lock in ['delete']:
         if method in ['patch', 'post']:
             raise LaymanError(error_code)
+    if method not in ['delete']:
```

Comment:
Missing `'wfst'` in `if method not in ['patch', 'delete']:` and `if current_lock not in ['patch', 'delete', 'post']:`